frontend/src/metabase/query_builder/components/NativeQueryEditor/CodeMirrorEditor/completers.tsx

```diff
@@ -64,12 +64,21 @@ export function useSchemaCompletion({ databaseId }: SchemaCompletionOptions) {
         return null;
       }
 
+      const seen = new Set();
+
+      const deduped = data.filter(([value]) => {
```

Comment:
const deduped = Array.from(new Set(data))?

Reply:
We are deduping by the first value of the tuple, not the whole tuple.